package.json

```diff
@@ -104,17 +105,18 @@
     "check-licenses": "wp-scripts check-licenses --production",
     "deploy": "grunt deploy",
     "dev": "wp-scripts start",
+    "diff-develop": "git diff develop --diff-filter=d --name-only",
     "env:start": "./bin/local-env/start.sh",
     "env:stop": "./bin/local-env/stop.sh",
     "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
     "lint": "npm-run-all --parallel lint:*",
-    "lint:css": "wp-scripts lint-style",
+    "lint:css": "npm run diff-develop | grep -E 'css$' | xargs wp-scripts lint-style",
     "lint:css:fix": "npm run lint:css -- --fix",
-    "lint:js": "wp-scripts lint-js",
+    "lint:js": "npm run diff-develop | grep -E '\\.js$' | xargs wp-scripts lint-js",
     "lint:js:fix": "npm run lint:js -- --fix",
-    "lint:php": "vendor/bin/phpcs",
+    "lint:php": "npm run diff-develop | grep -E '\\.php$' | xargs vendor/bin/phpcs",
```

Comment:
Maybe `grep -E '\\.php$'` is overkill as opposed to `grep php`

But `grep -E '\\.js$'` is probably better than simply `grep js`